src/bundle/Controller/Content/ContentTreeController.php

```diff
@@ -110,6 +134,117 @@ public function loadSubtreeAction(Request $request): Root
 
         return new Root($elements);
     }
+
+    /**
+     * @throws \Ibexa\AdminUi\Exception\InvalidArgumentException
+     * @throws \Ibexa\Contracts\Core\Repository\Exceptions\BadStateException
+     * @throws \Ibexa\Contracts\Core\Repository\Exceptions\InvalidArgumentException
+     * @throws \Ibexa\Contracts\Core\Repository\Exceptions\NotFoundException
+     * @throws \Ibexa\Contracts\Core\Repository\Exceptions\UnauthorizedException
```

Comment:
Is unauthorized really thrown by this method?

Reply:
Actually, after removing `loadLocation` it is not longer thrown.